packages/remix-testing/jest.config.js

```diff
@@ -4,5 +4,5 @@ module.exports = {
   displayName: "testing",
   setupFiles: [],
   testEnvironment: "jsdom",
-  setupFilesAfterEnv: ["@testing-library/jest-dom", "./jest.setup.js"],
+  setupFilesAfterEnv: ["./jest.setup.js", "@testing-library/jest-dom"],
```

Comment:
The order here matters due to testing-library setting up globals badly if they don't exist.